static/app/components/roleSelectControl.tsx

```diff
@@ -31,7 +38,10 @@ function RoleSelectControl({roles, disableUnallowed, ...props}: Props) {
             ({
               value: r.id,
               label: r.name,
-              disabled: disableUnallowed && !r.isAllowed,
+              disabled:
+                disableUnallowed &&
```

Comment:
Should these be && or || 🤔

Reply:
i believe && is fine, we want to disable if its not allowed and if its not a member invite or the inviting roll is member. demorgan's law makes the && valid _i think_.